Frontend/road-management/src/views/TariffView/TariffWindow.tsx

```diff
@@ -0,0 +1,260 @@
+                    name: selectedTariff.name,
+                    rates: selectedTariff.rates,
+                    roadIds: selectedTariff.roadIds
+                };
+
+                const tariffProxy = new TariffModelProxy();
+                // let proxyResponse = await tariffProxy.saveTariffdData(newtariff);
+
+                // const newtariffs = [...tariffList, newtariff];
+                // setTariffList(newtariffs);
+                // tariffDispatcher.showStatusMessage(proxyResponse);
+
+                // const fieldName = event.target[0].value;
+                // const fieldValue = event.target[1].value;
+                // console.log(fieldName)
+                // console.log(fieldValue);
+
+                // newtariff.rates = new Map(Object.entries(newtariff.rates)).set(fieldName, +fieldValue);
+                // let proxyResponse = await tariffProxy.updateTariff(newtariff);
+                // tariffDispatcher.showStatusMessage(proxyResponse);
+            };
+        }
+        onRoadIdSelected = (item: string | null) => {
+            if (item !== null) {
+                const tl = tariffList.find((tl) => tl.id!.toString() === item);
+                console.log(tl)
+                tl!.isValid = true;
+                setSelectedTariff(tl);
+            }
+        };
+
+    }
+
+    const tariffPresenter = new TariffPresenter();
+
+    return (
+        <div style={{ textAlign: "center" }}>
+            <h1 style={{ textAlign: "center" }}>Taryfikator</h1>
+            <DropdownButton style={{ marginTop: "20px" }}
+                variant={"info"}
+                title={"Droga"}
+                onSelect={tariffPresenter.onRoadIdSelected}
+            >
+                {tariffList.map((tl) => {
+                    return (
+                        <Dropdown.Item key={tl.id} eventKey={tl.id}>
+                            {tl.roadIds.join(',')}
+                        </Dropdown.Item>
+                    );
+                })}
+            </DropdownButton>
+            <Form>
+                <Form.Check reverse style={{ marginRight: "35px" }}
+                    type="checkbox"
+                    label={`Edytuj`}
+                    onClick={() => setEditing(!editing)}
+                />
+            </Form>
+            {editing &&
+                <>
+                    <Button
+                        onClick={() => setEditingField(!editingField)}
+                    >Edytuj
+                    </Button>
+                </>
+
+            }
+            <Form className="d-flex justify-content-center" style={{ margin: "30px" }}>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Kategoria samochodu</th>
+                            <th>Cena za kilometr</th>
+
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {selectedTariff !== undefined &&
```

Comment:
Jak te rates ma być możliwe do powiększania to tu przydałby się jakiś selectedTariff.rates.map() do tych inputów